ps_checkout.php

```diff
@@ -138,6 +138,7 @@ public function install()
     {
         // Install for both 1.7 and 1.6
         $defaultInstall = parent::install() &&
+            (new PrestaShop\AccountsAuth\Installer\Install())->installPsAccounts() &&
```

Comment:
Regarding the comments from PHPStan, it seems the PHP library needs to be added.

Reply:
What do you mean by adding the library? Add it via making a composer install before execute phpstan?

Reply:
This may require a `composer require` to be done, with the composer.json & composer.lock commited.